setup.py

```diff
@@ -33,7 +33,7 @@
     requirements.append("enum34==1.1.6")  # v1.1.6 is needed by halo 0.0.30 / spinners 0.0.24
     requirements.append("halo==0.0.30")  # halo==0.0.30 is the last version to support py2.7
     requirements.append("vivisect==0.1.0")
-    requirements.append("viv-utils")
+    requirements.append("viv-utils==0.3.18")
```

Comment:
Should we use `>=` instead of `==`? 🤔

Reply:
i think its better to be explicit here, esp with dependabot configured

Reply:
agreed, only need a PyPi release